File: listas de exercicio/listas 1/lista casa python/questao 5.py
kg = input('Digite seu peso em quilos:')
kg = float(kg)
A = input('Digite sua altura em cm:')
A = float(A)
B = kg / A ** 2
B = float(B)
IMC = B * 10000
input(f'seu IMC é igual a {IMC:.1f} um IMC normal é perto de 18')

File: listas de exercicio/listas 1/lista casa C/questao 5.c
#include <stdio.h>
#include <math.h>

int main(){
    float kg, A, IMC;

    printf("Digite seu peso em quilos:");
    scanf("%f", &kg);
    printf("Digite sua altura em cm:");
    scanf("%f", &A);

    IMC = (kg / pow(A, 2)) * 10000;

    printf("seu IMC é igual a %.2f um IMC normal é perto de 18", IMC);
}